swift-ticketing/test/swift_ticketing/db/booking_test.clj

```diff
@@ -0,0 +1,55 @@
+(ns swift-ticketing.db.booking-test
+  (:require [clojure.test :refer [deftest testing is use-fixtures]]
+            [swift-ticketing.fixtures :as fixtures]
+            [swift-ticketing.db.booking :as db-booking]
+            [swift-ticketing.factory :as factory]))
+
+(use-fixtures :each fixtures/clear-tables)
+
+(deftest insert-booking-test
+  (let [{:keys [db-spec test-user-id]} fixtures/test-env
+        expected {:booking-id (random-uuid)
+                  :user-id test-user-id
+                  :booking-status (factory/random-booking-status)}]
+    (testing "Insert booking query"
+      (db-booking/insert-booking
+       db-spec
+       (:user-id expected)
+       (:booking-id expected)
+       (:booking-status expected))
+      (let [booking (db-booking/get-booking db-spec (:booking-id expected))]
```

Comment:
The purpose of this unit test is to verify that the call to `insert-booking` created a row in the database. Calling `get-booking` to assert that is fragile, as the code might have just stored it in memory. Execute a SQL query from the unit test to assert that a row was created.